test/Renci.SshNet.IntegrationTests/SftpClientTests.cs

```diff
@@ -34,21 +34,21 @@ public void Create_directory_with_contents_and_list_it()
             Assert.IsTrue(_sftpClient.Exists(testFilePath));
 
             // Check if ListDirectory works
-            var files = _sftpClient.ListDirectory(testDirectory);
+            var expectedFiles = new List<(string FullName, bool IsRegularFile, bool IsDirectory)>()
+            {
```

Comment:
I'm aware that you do not agree with this, but please indent the braces.

Reply:
I also don't like using `var`, but I don't mind ;-)